app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -19,6 +21,9 @@ case class Project(
   description: String,
   visibility: Visibility,
   tileVisibility: Visibility,
+  isAOIProject: Boolean,
+  aoiCadence: Long, /* Milliseconds */
```

Comment:
I think it necessarily is an illegal state to have `none` as the cadence -- it wouldn't be checked then, but maybe there would be a way to invoke the check ad hoc in that case.